banking_proxy/lib/bunq.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Bunq do
+  @moduledoc """
+  API for interacting with the Bunq bank.
+  """
+  @behaviour BankingProxy.BankingBackend
+
+  use Agent
+  alias Bunq.API
```

Comment:
@vloothuis I was searching for this Bunq.API module but could not find it. Did I miss something?

Reply:
GitHub collapsed it because it is a large file. Therefore searching on the page did not yield the result. It's there if you look for `bunq/api`.